config/dev.json

```diff
@@ -95,5 +95,6 @@
   "openid": {
     "keyFile": "../config/key.json",
     "key": {}
-  }
+  },
+  "fxaEnv": "development"
```

Comment:
from meeting: let's make a new env variable for this feature and enable it in fxa-dev.